openapi-v3.yaml

```diff
@@ -1249,7 +1462,155 @@ components:
         created_at:
           type: "string"
           format: "date-time"
-          
+
+    License:
```

Comment:
Is this the right terminology here (`License`, `Group`, `PermissionStatement`, `GroupPermission`)?

Reply:
If I am understanding this correctly, these roughly correspond to the `user_license`, `user_group`, 'auth_permission`, and `auth_group_permissions` table respectively.

@parose I think we typically use `User License`, `User Group`, and `Permission Sets` - is that correct? 😬 I think `Group Permission` is good though.

Reply:
`User License` and `Group Permission` look good to me!

I think just `Group` is good here - it looks like this object is referring to the group itself (eg. the `groups` table), rather than the association between a user and a group, which is what we usually refer to as `user_group` (at least internally)

Off the top of my head I can't think of any place in dbt Cloud where we publicly name or display `PermissionStatements`, but internally I think they're usually referred to as either PermissionStatements or just Permissions, so this looks good too!

Reply:
Oof!  Goodness, yeah... it's definitely not a representation of the association itself.😆   Thank you for catching that Andrew!

How do Permissions differentiate between Permission Sets?